src/default_panels/GraphTransformsPanelDev.js

```diff
@@ -0,0 +1,102 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {
+  Radio,
+  // TransformAccordion,
+  TraceAccordion,
+  DataSelector,
+  Dropdown,
+  PlotlySection,
+  Numeric,
+} from '../components';
+import TransformAccordion from '../components/containers/TransformAccordionDev';
+import {connectAggregationToTransform} from '../lib';
+
+const AggregationSection = connectAggregationToTransform(PlotlySection);
+
+export class Aggregations extends Component {
+  render() {
+    const {
+      fullContainer: {aggregations = []},
+    } = this.context;
+    const {localize: _} = this.context;
+    if (aggregations.length === 0) {
+      return null;
+    }
+
+    return aggregations
+      .filter(aggr => aggr.target.match(/transforms\[\d*\]\./gi) === null)
+      .map(({target}, i) => (
+        <AggregationSection show key={i} aggregationIndex={i}>
+          <Dropdown
+            attr="func"
+            label={target}
+            options={[
+              {label: _('Count'), value: 'count'},
+              {label: _('Sum'), value: 'sum'},
+              {label: _('Average'), value: 'avg'},
+              {label: _('Median'), value: 'median'},
+              {label: _('Mode'), value: 'mode'},
+              {label: _('RMS'), value: 'rms'},
+              {label: _('Standard Deviation'), value: 'stddev'},
+              {label: _('Min'), value: 'min'},
+              {label: _('Max'), value: 'max'},
+              {label: _('First'), value: 'first'},
+              {label: _('Last'), value: 'last'},
+            ]}
+            clearable={false}
+          />
+        </AggregationSection>
+      ));
+  }
+}
+
+Aggregations.plotly_editor_traits = {no_visibility_forcing: true};
+Aggregations.contextTypes = {
+  fullContainer: PropTypes.object,
+  localize: PropTypes.func,
+};
+
+const GraphTransformsPanel = (props, {localize: _}) => {
+  return (
+    <TraceAccordion>
+      <TransformAccordion>
+        <Radio
+          attr="enabled"
+          options={[
+            {label: _('Enabled'), value: true},
+            {label: _('Disabled'), value: false},
+          ]}
+        />
+
+        <DataSelector label={_('By')} attr="groups" />
+
+        <DataSelector label={_('Filter')} attr="target" />
+
+        <Dropdown
+          attr="operation"
+          label={_('Operation')}
+          options={[
```

Comment:
well, I know that you know, but just make sure, 
there's more options than just those right, so i'll look for those in your next prs!